README.md

```diff
@@ -20,19 +22,29 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
 
 ![Web App Log Stream](resources/WebApp_LogStream.png)
 
+## Configuring Zipdeploy
+
 * How to configure zip deployment:
-  * If you set the deployment source of your web app to “None”, the deploy command will Zip the contents of a selected folder and upload the Zip file to Azure. You might want to use the following settings to customize which files to include/exclude:
+  * If you set the deployment source of your web app to “None” (the default source on app creation), the deploy command will Zip the contents of a selected folder and upload the Zip file to Azure.
+
+  * VS Code will prompt on deploy if you would like to configure your project for faster deployment.  If you click "Yes", your project's settings will be changed in the following:
+    * 'appService.zipIgnorePattern' will include build artifacts to be ignored for zipping.  These will now be built on the server by running the appropriate build command.
+    * A '.deployment' file will be created in the root of the project.  This file configures 'SCM_DO_BUILD_DURING_DEPLOYMENT=true,' enabling build on deploy.
+      > NOTE: Currently only 'node' runtimes support this feature.
+  * If you select 'Never show again,' the 'appService.showBuildDuringDeployPrompt' setting be set to 'false' and you will no longer be prompted for this project.  Delete this setting or set it to 'true' to re-enable the prompt.
+
+  ### Zipdeploy Configuration Settings
 
   * 'appService.zipGlobPattern'
     * Uses a glob pattern to define which files to be included in the deployment. The default value is “**/*”.
 
   * 'appService.zipIgnorePattern'
-    * Uses a glob pattern to define which files to be excluded from the deployment. The default value is “” which doesn’t exclude any files/folders.
+    * Uses a glob pattern to define which files to be excluded from the deployment. The default value is [] which doesn’t exclude any files/folders.
 
   * For example, you might want to exclude the “node_modules” folder from the deployment to speed up the Zip file creation and uploading. In this case, you will need the following setting:
-    * “appService.zipIgnorePattern”: “node_modules{,/**}”
+    * “appService.zipIgnorePattern”: [“node_modules{,/**}”]
   * And in order to have the web app run proper deployment command to restore the npm packages, you need to have the following Application Setting on your site:
```

Comment:
In order to have the web app restore npm packages as part of deployment, you need ...

Reply:
?